cookiecutter/{{cookiecutter.github_repo_name}}/.github/workflows/auto-dependabot.yaml

```
@@ -0,0 +1,22 @@
+name: Auto-merge Dependabot PRs
```

Nitpick: Use singular, as this approves and merges PRs one-by-one? It might be more clear what it does when looking at the GitHub "Actions" tab.


```suggestion
name: Auto-merge Dependabot PR
```